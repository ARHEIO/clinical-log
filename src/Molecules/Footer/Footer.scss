.app-footer {
  align-items: center;
  display: flex;
  flex-direction: column;
  flex-shrink: 0;
  justify-content: center;
  margin-top: auto;
  padding: 16px;
  background-color: #87AEB5;

  & > * {
    @media (min-width: 960px) {
      width: 800px;
    }
  }
}
