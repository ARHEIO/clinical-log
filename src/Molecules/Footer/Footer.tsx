import React, { ReactElement } from 'react';
import './Footer.scss';

const Footer = (): ReactElement => (
  <footer className="app-footer">
    <p>
      Made as part of a coding challenge
    </p>
  </footer>
);

export default Footer;
