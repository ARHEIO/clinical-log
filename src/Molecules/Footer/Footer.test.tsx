import React from 'react';
import { render } from '@testing-library/react';
import pretty from 'pretty';
import Footer from './Footer';

it('renders learn react link', () => {
  const { container } = render(<Footer />);
  expect(pretty(container.innerHTML)).toMatchSnapshot();
});
