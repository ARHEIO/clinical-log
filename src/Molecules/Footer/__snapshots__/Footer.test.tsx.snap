// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
"<footer class=\\"app-footer\\">
  <p>Made as part of a coding challenge</p>
</footer>"
`;
